app/models/poem.rb

```diff
@@ -1,5 +1,8 @@
 class Poem < ApplicationRecord
+  POEM_STATUSES = ["Rough Draft", "Ready", "Submitted", "Accepted", "Published"]
```

Comment:
You'll still want to freeze the array even if you have `# frozen_string_literal: true`